aspnetcore/client-side/libman/libman-vs.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Use LibMan with ASP.NET Core in Visual Studio
+author: scottaddie
+description: Learn how to use LibMan in an ASP.NET Core project with Visual Studio.
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 08/13/2018
+uid: client-side/libman/libman-vs
+---
+# Use LibMan with ASP.NET Core in Visual Studio
+
+By [Scott Addie](https://twitter.com/Scott_Addie)
+
+Visual Studio has built-in support for LibMan in ASP.NET Core projects, including:
+
+* Support for configuring and running LibMan restore operations on build
+* Menu items for triggering LibMan restore and clean operations
+* Search dialog for finding libraries and adding the files to your project
+* Editing support for *libman.json*&mdash;the LibMan manifest file
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/client-side/libman/samples/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Prerequisites
+
+* Visual Studio 2017 version 15.8 or later with the **ASP.NET and web development** workload
+
+## Add library files
+
+Library files can be added to an ASP.NET Core project in two different ways:
+
+1. [Use the Add Client-Side Library dialog](#use-the-add-client-side-library-dialog)
+1. [Edit the LibMan manifest](#edit-the-libman-manifest)
+
+### Use the Add Client-Side Library dialog
+
+Follow these steps to install a client-side library:
+
+* In **Solution Explorer**, right-click the project folder in which the file(s) should be added. Choose **Add** > **Client-Side Library**. The **Add Client-Side Library** dialog appears:
+
+  ![Add Client-Side Library dialog](_static/add-library-dialog.png)
+
+* Select the library provider from the **Provider** drop down. CDNJS is the default provider.
+* Type the library name to fetch in the **Library** text box. IntelliSense provides a list of libraries beginning with the provided text.
+* Select the library from the IntelliSense list. Notice the library name is suffixed with the `@` symbol and the latest stable version known to the selected provider.
+* Decide which files to include:
+  * Select the **Include all library files** radio button to include all of the library's files.
+  * Select the **Choose specific files** radio button to include a subset of the library's files. When the radio button is selected, the file selector tree becomes usable. Check the boxes to the left of the file names to download.
+* Specify the project folder, for storing the files, in the **Target Location** text box. As a recommendation, store each library in a separate folder.
```

Comment:
Comma splices ... but if you want that "for storing the files" provided parenthetically, then I recommend going with parentheses ...

> Specify the project folder (for storing the files) in the ...

Could really just drop both commas tho. It's really fine just as ...

> Specify the project folder for storing the files in the ...

I've been advised in the past that we can say "we recommend" whenever needed ... for example ....

> We recommend storing each library in a separate folder.